src/driver.jl

```diff
@@ -260,6 +260,12 @@ const __llvm_initialized = Ref(false)
         end
     end
 
+    for (name, plugin) in PLUGINS
+        if plugin.finalize_module !== nothing
+            plugin.finalize_module(job, compiled, ir)
+        end
+    end
```

Comment:
@wsmoses what other information would you need, purely to annotate the functions for a later optimization pass?